x-pack/plugins/ml/public/components/anomalies_table/_anomalies_table.scss

```diff
@@ -75,9 +75,25 @@
     min-width: 150px;
   }
 
+  .euiLink {
```

Comment:
Try not to directly manipulate any EUI class styles. The best thing to do here is override with a custom class you pass to  the component like so:

`<EuiLink className="customClassName" />`

Reply:
Thanks! Fixed by: 2ada3cd191ca449d4b87d216591050e00ae41a0f